### PR TITLE
Modernized with Embulk v0.10.28 styles

### PR DESCRIPTION
**Pull request by minidragon88**

### CODE REVIEW

File: gradle/dependency-locks/embulkPluginRuntime.lockfile [hunk 1]
```diff
@@ -1,30 +0,0 @@
-# This is a Gradle generated file for dependency locking.
```
Comment (vietnguyen-td) [inline gradle/dependency-locks/embulkPluginRuntime.lockfile]: could you re-add this file for tracking dependency with core?

**Review by vietnguyen-td — APPROVED** [review]
LGTM

**Review by nukala — APPROVED** [review]
Reviewed for Data Exfiltrations

File: gradle/dependency-locks/embulkPluginRuntime.lockfile (hunk 1 above)
Comment (minidragon88) [reply gradle/dependency-locks/embulkPluginRuntime.lockfile]: Done

**Pull request merged**